utilities/tools/platform/Set-ApiSpecsFile.ps1

```diff
@@ -0,0 +1,49 @@
+﻿<#
+.SYNOPSIS
+Update the API Specs file in the given path
+
+.DESCRIPTION
+Update the API Specs file in the given path. The file contains an outline of all Provider Namespaces with their Resource Types and supported API versions.
+
+.PARAMETER SpecsFilePath
+Optional. The path the the file to create/overwrite. By default points to path '<root>/utilities/src/apiSpecsList.json'
+
+.EXAMPLE
+Set-ApiSpecsFile -SpecsFilePath 'C:/dev/ResourceModules/utilities/src/apiSpecsList.json'
+
+Update the file in path 'C:/dev/ResourceModules/utilities/src/apiSpecsList.json' with the latest API versions.
+#>
+function Set-ApiSpecsFile {
+
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    param (
+        [Parameter(Mandatory = $false)]
+        [string] $SpecsFilePath = (Join-Path (Split-Path (Split-Path $PSScriptRoot -Parent)) 'src' 'apiSpecsList.json')
+    )
+
+    # Install and or import module
+    if (-not (Get-Module 'AzureAPICrawler' -ListAvailable)) {
+        if ($PSCmdlet.ShouldProcess("Module 'AzureAPICrawler with version [0.2.0]'", 'Install')) {
+            $null = Install-Module 'AzureAPICrawler' -Scope 'CurrentUser' -Repository 'PSGallery' -RequiredVersion '0.2.0' -Force
```

Comment:
I'd suggest to move the version to an input parameter and set 0.2.0 as default value

Reply:
Done

Reply:
Done but not used
```suggestion
            $null = Install-Module 'AzureAPICrawler' -Scope 'CurrentUser' -Repository 'PSGallery' -RequiredVersion $ModuleVersion -Force
```

Reply:
done